src/zrc/datamodel/models.py

```diff
@@ -186,8 +188,14 @@ class Zaak(APIMixin, models.Model):
     )
 
     relevante_andere_zaken = ArrayField(
-        models.URLField(_("URL naar andere zaak"), max_length=1000),
-        blank=True, default=list
+        JSONField(blank=True),
```

Comment:
JSONField already supports arrays as structures, so making an `ArrayField` of `JSONField` is a weird construction. JSONField is also too broad for this purpose - a better fit would be `HStoreField`.

Still, my suggestion would be to define a model to hold an "externe zaak" with its own, normalized fields and set up FK to `zaak` for this.